jinseok/service/userService.js

```diff
@@ -27,6 +38,50 @@ const signUp = async (
   return createUser;
 };
 
+const login = async (email, password) => {
+  emailValidationCheck(email);
```

Comment:
```suggestion
  await emailValidationCheck(email);
```
내부 로직이 복잡하거나 수행 시간이 오래걸리는 함수가 아니기 때문에 정상 동작에는 문제 없을 수 있으나, 보다 확실한 비동기 처리를 위해 await를 추가해주세요.